source/common/http/http2/codec_impl.h

```diff
@@ -73,14 +73,26 @@ class Utility {
                                           HeaderString& cookies);
 };
 
+struct SettingsEntryHash {
```

Comment:
Per my other comment wondering if we can get rid of the map as the number of elements will be tiny but up to you.

Reply:
This has been moved to a test utility class.